### DISCUSSION TITLE
FormDialog 如何设置宽高等样式呢

### DISCUSSION THREAD

**Discussion by Zwahxf** [alibaba/formily#2945] [Q&A 2.x] [answered]
FormDialog 如何设置宽高等样式呢

**Comment by janryWang** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/4060976/158581937-40eec701-6d85-48dc-9332-3e25ba70905a.png)

**Reply by Zwahxf** [reply]
title中ImodalProps没有设置的对应参数，renderer中是对modal中的component组件样式的设置，但是弹出的modal宽高在哪里设置呢

**Reply by janryWang** [reply]
modalProps参考antd

**Comment by relines** [comment]
const dialog = FormDialog({
    title: '新建',
    width: '80%'
  }, () => {})
试了下这样写，文档写的不够清晰，还是得试。
